### PR TITLE
Support for el9

### PR DESCRIPTION
**Pull request by andrea-manzi**
# Summary

Support for el9, packages not available are conditionally excluded in spec file.

---

#4

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -41,3 +63,20 @@ jobs:
           yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
           yum install -y http://repository.egi.eu/sw/production/umd/4/centos7/x86_64/updates/umd-release-4.1.3-1.el7.centos.noarch.rpm
           yum localinstall -y ui-*.rpm
+
+
```
Comment (gwarf) [inline .github/workflows/build.yml]:
```suggestion
```

**Review by gwarf — CHANGES_REQUESTED** [review]
LGTM.
Maybe it would be better to use dnf everywhere for AlmaLinux9

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -41,3 +63,20 @@ jobs:
...
+    runs-on: ubuntu-latest
+    container: almalinux:9
+    steps:
+      - uses: actions/download-artifact@v3
+        with:
+          name: rpms9
+      - name: Install generated RPMs
+        run: |
+          dnf install -y epel-release
+          dnf install -y https://research.cs.wisc.edu/htcondor/repo/23.x/htcondor-release-current.el9.noarch.rpm
```
Comment (gwarf) [inline .github/workflows/build.yml:80]: Maybe add a FIXME there, to switch to UMD5, if/when possible

File: ui.spec [hunk 3]
```diff
@@ -74,6 +80,8 @@ rm -rf %{buildroot}
 %doc /usr/share/doc/ui/README.md
 
 %changelog
+* Tue Jun 04 2024 <andrea.manzi@egi.eu> - 6.0.0-1
+- Support for El9
```
Comment (gwarf) [inline ui.spec]:
```suggestion
- Support for El9 (#6 ) (Andrea Manzi)
```

**Comment by gwarf** [comment]
You can also already update the CHANGELOG file, as you added a new version to the spec

**Comment by andrea-manzi** [comment]
@gwarf thanks for the suggestions, going to apply them now

File: .github/workflows/build.yml [hunk 4]
```diff
@@ -41,3 +63,21 @@ jobs:
...
+    needs: build-almalinux9
+    runs-on: ubuntu-latest
+    container: almalinux:9
+    steps:
+      - uses: actions/download-artifact@v3
+        with:
+          name: rpms9
+      - name: Install generated RPMs
+        run: |
+          #FIXME remove external repo when UMD5 is available
```
Comment (gwarf) [inline .github/workflows/build.yml]:
```suggestion
          # FIXME: remove external repo when UMD5 is available 
```

File: CHANGELOG [hunk 5]
```diff
@@ -7,6 +7,8 @@ and this project adheres to
 [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
```
Comment (gwarf) [inline CHANGELOG:9]:
```suggestion
## [Unreleased]

```

File: ui.spec [hunk 6]
```diff
@@ -74,6 +80,8 @@ rm -rf %{buildroot}
 %doc /usr/share/doc/ui/README.md
 
 %changelog
+* Tue Jun 04 2024 <andrea.manzi@egi.eu> - 6.0.0-1
+- Support for El9 (#6 ) (Andrea Manzi)
```
Comment (gwarf) [inline ui.spec]:
```suggestion
- Support for El9 (#6) (Andrea Manzi)
```

**Review by gwarf — CHANGES_REQUESTED** [review]
Thanks, sorry just a last few minor things

**Review by gwarf — APPROVED** [review]
LGTM, thanks!

**Pull request merged**